.tool-versions

```diff
@@ -1 +1 @@
-golang 1.21.9
+golang 1.22.5
```

Comment:
~~1.22.6 is newer, perhaps use that?~~ I see the `brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.22` is at `go version go1.22.5 (Red Hat 1.22.5-1.el9) linux/amd64`, makes sense to be on the same patch version.

I guess we could enforce that all versions used here are matched in CI somehow? Would that be something of value?

Reply:
Yeah. Not sure if we can build the project if the version on the builder image is older (even just at the patch version). Need to play with it further.

Reply:
Following up in https://github.com/enterprise-contract/ec-cli/pull/1917.

> I guess we could enforce that all versions used here are matched in CI somehow? Would that be something of value?

Maybe let's see if not doing that causes any issues?